# Example parameters (customize as needed)
pan_increment: 30.0
tilt_increment: 30.0
pan_range: [-90.0, 90.0]
tilt_range: [-90.0, 90.0]
image_topic: "/rgb_stereo_publisher/color/image"
depth_topic: "/rgb_stereo_publisher/stereo/depth"
pan_topic: "/mastcam_pan_controller/command"
tilt_topic: "/mastcam_tilt_controller/command"
